rasgotransforms/rasgotransforms/transforms/plot/plot_test.py

```diff
@@ -0,0 +1,215 @@
+import unittest
```

Comment:
Organization recommendation: we may want move this and future test files into the `/tests/` folder.

Since this is a public repo, there's value in keeping the tests separate from the transform library